java/java-ranger-regression/WBS/prop1/Main.java

```diff
@@ -0,0 +1,18 @@
+import org.sosy_lab.sv_benchmarks.Verifier;
+public class Main {
+	
+	public static void main(String[] args) {
+		WBS wbs= new WBS();
+		int PedalPos;
+		boolean AutoBrake, Skid;
+		for (int i = 0; i < 2; i++) {
+		  PedalPos = Verifier.nondetInt();
+		  AutoBrake = Verifier.nondetBoolean();
+		  Skid = Verifier.nondetBoolean();
+		  wbs.update(PedalPos, AutoBrake, Skid);
+		  // This assertion should fail:
```

Comment:
This comment is contradicting the expected verdict. Please remove the comment line or change the label accordingly.

Reply:
Fixed. Thanks for catching this.